app/views/feeds/clean.builder

```diff
@@ -8,6 +8,7 @@ xml.rss "version" => "2.0", "xmlns:dc" => "http://purl.org/dc/elements/1.1/" do
    xml.description "Recently posted maps at MapKnitter.org, a Public Lab open source research initiative"
 
    @maps.each do |map|
+     map.body = "<p><img src='" map.warpables.first.image.url(:medium) "' alt='" map.name"'/></p>"
```

Comment:
This looks good! However, I think the change needs to happen further down on line 20, modifying `xml.description map.description.to_s + " " + map.warpables.first.image.url(:medium) + spamlink`

Would you mind making this change? Thanks so much!